dm/dm/worker/source_worker.go

```diff
@@ -798,35 +799,43 @@ func (w *SourceWorker) handleSubTaskStage(ctx context.Context, stageCh chan ha.S
 // operateSubTaskStage returns TaskOp.String() additionally to record metrics.
 func (w *SourceWorker) operateSubTaskStage(stage ha.Stage, subTaskCfg config.SubTaskConfig) (string, error) {
 	var op pb.TaskOp
-	switch {
-	case stage.Expect == pb.Stage_Running, stage.Expect == pb.Stage_Paused:
```

Comment:
why deleted the `stage.Expect == pb.Stage_Paused` case?

Reply:
no need this check now, seems we do not create a task that in `Paused` state？

Reply:
if a task is paused and DM-worker does failover, it will see Paused status.

Reply:
added in https://github.com/pingcap/tiflow/pull/4510/commits/d7b4e7da8cef40069589e9e09a9c09eba4a57e53